stem_cell/tests/test_differentiator.py

```diff
@@ -39,6 +49,12 @@ def get_version_control_token(self):
     def set_version_control_token(self, token):
         self.version_control_token = token
 
+    def set_version_control_private_key(self, private_key):
```

Comment:
I think these are only simple get methods in test scenario, these are mocks... they must be methods because the production classes are performing a calculation